package.json

```diff
@@ -148,6 +148,9 @@
     "prettier": "^3.3.3",
     "rollup-plugin-visualizer": "^5.6.0",
     "storybook": "^8.3.2",
+    "@storybook/addon-themes": "^8.3.2",
+    "@storybook/blocks": "^8.3.2",
+    "add": "^2.0.6",
```

Comment:
Where are we using this package?

Reply:
Oh, yeah this should for sure be removed. Thank you 🤦